pkg/blobstore/readfallback/read_fallback_blob_access.go

```diff
@@ -90,5 +91,23 @@ func (ba *readFallbackBlobAccess) FindMissing(ctx context.Context, digests diges
 	if err != nil {
 		return digest.EmptySet, util.StatusWrap(err, "Secondary")
 	}
+
+	// Replicate the blobs that are present only in the secondary backend to primary
```

Comment:
Nit: Please end all sentences in comments with a full stop (dot), and make sure they are wrapped to at most 80 columns.